federatedscope/core/worker/client.py

```diff
@@ -264,7 +264,12 @@ def callback_funcs_for_evaluate(self, message: Message):
             logging.info(
                 'Client #{:d}: (Evaluation (val set) at Round #{:d}) {:s} is {:.6f}'
                 .format(self.ID, self.state, key, val_metrics[key]))
-        metrics = {**test_metrics, **val_metrics}
+        train_metrics = self.trainer.evaluate(target_data_split_name='train')
```

Comment:
Is it possible to allow users to determine whether to conduct evaluation on a specific split?